modin/backends/pandas/query_compiler.py

```diff
@@ -1116,40 +1125,54 @@ def map_func(df):
         )
         result = self.__constructor__(new_modin_frame)
 
-        if is_all_multi_list:
+        def compute_index(index, columns, consider_index=True, consider_columns=True):
+            def compute_axis(axis):
```

Comment:
Please, change this name to more appropriate name

Reply:
fixed